charts/opentelemetry-operator/templates/NOTES.txt

```diff
@@ -3,6 +3,6 @@
 {{ end }}
 
 {{ $.Chart.Name }} has been installed. Check its status by running:
-  kubectl --namespace {{ .Release.Namespace }} get pods -l "release={{ $.Release.Name }}"
+  kubectl --namespace {{ .Release.Namespace }} get pods -l "app.kubernetes.io/name={{ $.Release.Name }}"
```

Comment:
can you also please bump the patch version of the operator Chart.yaml and rerun `make generate-examples CHARTS=opentelemetry-operator`

Reply:
i have run `make generate-examples CHARTS=opentelemetry-operator` and created a new version at my forked own branch,what can i do next

Reply:
i have done it please check